source/solvers/gls_nitsche_navier_stokes.cc

```diff
@@ -137,6 +147,20 @@ GLSNitscheNavierStokesSolver<dim, spacedim>::assemble_nitsche_restriction()
                         this->fe->shape_value(k, ref_q);
                     }
                 }
+
+              const double u_mag = velocity.norm();
+
+
+              const double tau =
+                this->simulation_control->get_assembly_method() ==
+                    Parameters::SimulationControl::TimeSteppingMethod::steady ?
+                  1. / std::sqrt(
+                         std::pow(2. * u_mag / h_cell, 2) +
+                         9 * std::pow(4 * viscosity / (h_cell * h_cell), 2)) :
+                  1. / std::sqrt(
+                         std::pow(sdt, 2) + std::pow(2. * u_mag / h_cell, 2) +
+                         9 * std::pow(4 * viscosity / (h_cell * h_cell), 2));
```

Comment:
Would it be useful to put the tau calculation in a common function, since we use it in a few classes? 
If we push even further, could we put the calculation of the stabilization terms in a separate class that any solver that we want to stabilize could use? Just an idea, maybe for a future hackathon

Reply:
Good idea. This would be a great follow-up PR.